cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -425,9 +426,11 @@ private boolean gatherxStepMinMax(double[] xs) {
         boolean xStepRangeIsConsistent = true;
         if (xStepMin < 0 || xStepMax <= 0 || xStepMin > xStepMax) {
             xStepRangeIsConsistent = false;
-            String reason = String.format("Inconsistent xStepMin, xStepMax [%f, %f]",
-                    xStepMin,
-                    xStepMax);
+            final double xStepMinParam = xStepMin;
+            final double xStepMaxParam = xStepMax;
+            Supplier<String> reason = () -> String.format("Inconsistent xStepMin, xStepMax [%f, %f]",
+                    xStepMinParam,
+                    xStepMaxParam);
```

Comment:
I don't understand this refactoring

Reply:
I think it may be because lambda only allows final constants. Not a great way to use it tho... Maybe with `double[]`?

Reply:
I miss something during my first review.